contracts/deprecated/CrossChain.sol

```diff
@@ -152,55 +152,7 @@ contract CrossChain is System, ICrossChain, IParamSubscriber {
         _;
     }
```

Comment:
please also check the following import, and delete unnecessary ones:
```
import "../interface/0.6.x/IApplication.sol";
import "../interface/0.6.x/ICrossChain.sol";
import "../interface/0.6.x/ITokenHub.sol";
import "../interface/0.6.x/ILightClient.sol";
import "../interface/0.6.x/IRelayerIncentivize.sol";
import "../interface/0.6.x/IRelayerHub.sol";
import "../interface/0.6.x/IBSCValidatorSetV2.sol";
import "../lib/0.6.x/Memory.sol";
import "../lib/0.6.x/BytesToTypes.sol";
import "../interface/0.6.x/IParamSubscriber.sol";
import "../System.sol";
import "../lib/0.6.x/MerkleProof.sol";
```

Reply:
We can delete unused const.

Reply:
The `EmergencyProposal` is no longer needed

Reply:
`EmergencyProposal ` is used in `mapping(bytes32 => EmergencyProposal) public emergencyProposals; `

Reply:
fixed